src/fidesops/ops/api/v1/endpoints/email_endpoints.py

```diff
@@ -97,10 +98,10 @@ def patch_config_by_key(
     try:
         return update_email_config(db=db, key=config_key, config=email_config)
     except EmailConfigNotFoundException:
-        logger.warning(f"No email config found with key {email_config.key}")
+        logger.warning(f"No email config found with key {config_key}")
         raise HTTPException(
-            status_code=HTTP_400_BAD_REQUEST,
-            detail=f"No email config found with key {email_config.key}",
+            status_code=HTTP_404_NOT_FOUND,
+            detail=f"No email config found with key {config_key}",
         )
 
     except Exception as exc:
```

Comment:
Added in an earlier PR, but I don't know if the 422 is the best response code here.  FastAPI throws these 422 here often as a result of failed validation, but if we make it to this point, it's probably more likely an issue on our end, rather than their end, so a 5xx might be more appropriate for this and the POST endpoint.

Reply:
that's a good point, I'll update